scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Ok.
What about changing it to bash then though anyway?
It's clearer in general, and allows for using `[[` etc.